modules/ai/assets/js/editor/pages/form-layout/hooks/use-attach-url-service.js

```diff
@@ -1,21 +1,21 @@
 import { useTheme } from '@elementor/ui';
 import { useState } from 'react';
-import useUserInfo from '../../../hooks/use-user-info';
+import { useRemoteConfig } from '../context/remote-config';
 
 export const useAttachUrlService = ( ) => {
 	const [ currentUrl, setCurrentUrl ] = useState( '' );
 	const theme = useTheme();
-	const { isLoaded, builderUrl } = useUserInfo();
+	const { isLoaded, isError, remoteConfig } = useRemoteConfig();
 
-	if ( ! isLoaded ) {
+	if ( ! isLoaded || isError || ! remoteConfig.webBaseBuilderUrl ) {
 		return {
 			iframeSource: '',
 			currentUrl,
 			setCurrentUrl,
 		};
 	}
 
-	const urlObject = new URL( builderUrl );
+	const urlObject = new URL( remoteConfig.webBaseBuilderUrl );
```

Comment:
not sure what this name means, but shouldn't it be "webBased" (with "d")?

```suggestion
	const urlObject = new URL( remoteConfig.webBasedBuilderUrl );
```